portfolio/templates/main/entry.html

```diff
@@ -52,138 +52,169 @@
-                        <a rel="nofollow noopener noreferrer" class="share-window" href="https://twitter.com/intent/tweet?text={{ page.title }}&url={{ page.full_url }}&via=ColumbiaCTL" aria-label="Share on Twitter"><i class="fa fa-twitter px-1" aria-hidden="true"></i></a>
-                        <a rel="nofollow noopener noreferrer" class="share-window" href="https://www.linkedin.com/shareArticle?mini=true&url={{ page.full_url }}&title={{ page.title }}&summary={{page.search_description}}"><i class="fa fa-google-plus px-1" aria-label="Share on LinkedIn"></i></a>
-                    </div>
-                    </div>
-                </div>
-            </section>
-
-            {% if page.overview %}
-            <section class="overview">
-            <p>{{page.overview}}</p>            
+    <section class="entry-section entry-section__metadata row">
+        <div class="col-5 py-2 px-0">
+            {% with disciplines=page.discipline.all %}
+            {% if disciplines %}
+            <dl class="d-flex flex-row m-0">
+                <dt class="pr-2 my-0">Discipline{{ disciplines|pluralize }}:</dt>
+                {% for discipline in disciplines %}
+                <dd class="pr-1 my-0">{{ discipline.name }}{% if not forloop.last %},{% endif %}</dd>
+                {% endfor %}
+            </dl>
+            {% endif %}
+            {% endwith %}
 
-            </section>
+            {% with award_types=page.award_type.all %}
+            {% if award_types %}
+            <dl class="d-flex flex-row m-0">
+                <dt class="pr-2 my-0">Provost Grant:</dt>
+                {% for award_type in award_types %}
+                <dd class="pr-1 my-0">{{ award_type.name }}{% if not forloop.last %}, {% endif %}</dd>
+                {% endfor %}
+            </dl>
             {% endif %}
+            {% endwith %}
+        </div>
+        <div class="col-4 py-2 px-0">
+             <dl class="d-flex flex-row m-0">
+                <dt class="pr-2 my-0">Released:</dt>
+                <dd class="pr-1 my-0">{{page.release_date|date:"F Y"}}</dd>
+            </dl>
+             <dl class="d-flex flex-row m-0">
+                <dt class="pr-2 my-0">Updated:</dt>
+                <dd class="pr-1 my-0">{{page.revision_date|date:"F Y"}}</dd>
+            </dl>
+        </div>
+        <div class="col-3 text-nowrap py-2 px-0">
+            {% if page.project_url %}
+                <a href="{{ page.project_url }}" target="_blank">
+                Visit the site
+                <i class="fa fa-external-link" aria-hidden="true"></i>
+                </a>
+            {% endif %}
+            {% if page.infosheet %}
+                | <a href="{{ page.infosheet.url }}" target="_blank">
```

Comment:
Per code scanning note, please add `rel="noopener noreferrer"` to links that open in a separate add.